docs/intro-react-native-components.md

```diff
@@ -0,0 +1,100 @@
+---
+id: intro-react-native-components
+title: Core Components and Native Components
+---
+
+React Native is an open source framework for building Android and iOS applications using [React](https://reactjs.org/) and the app platform’s native capabilities. With React Native, you use JavaScript to access your platform’s APIs as well as to describe the appearance and behaviour of your UI using React components, bundles of reusable, nestable code. You can learn more about React in the next section. But first, let's cover how components work in React Native.
+
+## Views and mobile development
+
+In Android and iOS development, a **view** is the basic building block of UI: a small rectangular element on the screen which can be used to display text, images, or respond to user input. Even the smallest visual elements of an app, like a line of text or a button, are kinds of views. Some kinds of views can nest other views. It’s views all the way down!
```

Comment:
Should "View" be capitalized here since I assume it is referring to the RN primitive?

Reply:
Maybe: "Some kinds of views can nest other views" -> "Some kinds of views can be composed of other views"?

Reply:
"Some kinds of views can be composed of other views" feels clunky. What about "Some kinds of views can contain other views?" It's a little more common language.

Reply:
Re: "view", here I was trying to think of "the views that exist in iOS and Android development. We haven't quite gotten to introducing React Native's Core Component with a capital V: View (not quite yet). Thoughts?

Reply:
I don't think this needs to be capitalized since it doesn't feel like referring to the React Native view, especially because it starts with "In Android and iOS development". 

I'm good with "can contain", it feels clearer than "can nest" or "can composed". 👍

Reply:
Thanks for clearing that up, Eli! Also, great job on your intro materials I stole from ;)